switchtec.c

```diff
@@ -1568,6 +1568,19 @@ static enum switchtec_rel get_release(struct switchtec_dev *stdev)
 	return SWITCHTEC_REL_UNKNOWN;
 }
 
+static enum switchtec_evlist_ver get_evlist_ver(struct switchtec_dev *stdev)
+{
```

Comment:
This seems rather unnecessary now.  The evlist is the same as the release.

Reply:
Will remove. Do you think we need to check the release when probe? Like the driver currently support firmware releases through MR4, if customer use it on MR5, we quit in probe, or just assume the MR5 is supported?